code/modules/mob/living/carbon/death.dm

```diff
@@ -24,7 +24,7 @@
 	M.Scale(1.8, 1.2)
 	animate(src, time = 40, transform = M, easing = SINE_EASING)
 
-/mob/living/carbon/gib(no_brain, no_organs, no_bodyparts, safe_gib = FALSE)
+/mob/living/carbon/gib(no_brain, no_organs, no_bodyparts, safe_gib = TRUE)
```

Comment:
Again, please stop shoving this into random PRs without talking to anyone about it. I'm not interested in having extreme explosions just blow your limbs off like a lego man

Reply:
additionally if your going to make the change because you want to prevent it specifically for explosion related gibbing it needs to be for those only instead of turning most if not all gibs into safe ones.
make this its own pr please..... and actually elaborate why..